assets/ingredients/ingredients.json

```diff
@@ -15720,7979 +15720,8254 @@
         "C_BETA_CRYPT_MCG": 0
     },
     "CHEESE,BAKED": {
-        "R_Code": 1,
-        "R_DryM": 1,
-        "R_Water": 1,
-        "R_Energy": 1,
-        "R_Protein": 1,
-        "R_Lipid": 1,
-        "R_Cho": 1,
-        "R_Fiber": 1,
-        "R_Ca": 1,
-        "R_Fe": 1,
-        "R_Zn": 1,
-        "R_VitC": 0.65,
-        "R_Thiamin": 0.75,
-        "R_Riboflavin": 1,
-        "R_Niacin": 1,
-        "R_VitB_6": 0.75,
-        "R_Folate_food": 0.8,
-        "R_FolicAcid": 0.8,
-        "R_Folate_total": 0.8,
-        "R_Folate_DFE": 0.8,
-        "R_VitB_12": 0.55,
-        "R_VitA_IU": 1,
-        "R_VitA_RE": 1,
-        "R_Retinol": 1,
-        "R_AlphaCarotene": 1,
-        "R_BetaCarotene": 1,
-        "R_BetaCryptoxanthin": 1
+        "C_CODE": 1,
+        "C_STATE": 1,
+        "C_DRYMATTER_G": 1,
+        "C_WATER_G": 1,
+        "C_ENERG_KCAL": 1,
+        "C_PROTEIN_G": 1,
+        "C_LIPID_TOT_G": 1,
+        "C_CARBOHYDRT_G": 1,
+        "C_FIBER_TD_G": 1,
+        "C_CALCIUM_MG": 1,
+        "C_IRON_MG": 1,
+        "C_ZINC_MG": 1,
+        "C_VIT_C_MG": 0.65,
+        "C_THIAMIN_MG": 0.75,
+        "C_RIBOFLAVIN_MG": 1,
+        "C_NIACIN_MG": 1,
+        "C_VIT_B6_MG": 0.75,
+        "C_FOLATE_TOT_MCG": 0.8,
+        "C_FOLIC_ACID_MCG": 0.8,
+        "C_FOOD_FOLATE_MCG": 0.8,
+        "C_FOLATE_MCG_DFE": 0.8,
+        "C_VIT_B12_MCG": 0.55,
+        "C_VIT_A_IU": 1,
+        "C_VIT_A_MCG_RAE": 1,
+        "C_RETINOL_MCG": 1,
+        "C_ALPHA_CAROT_MCG": 1,
+        "C_BETA_CAROT_MCG": 1,
+        "C_BETA_CRYPT_MCG": 1
     },
```

Comment:
What exactly has been changed here? And what do the standardized fields mean now in short?

Reply:
Basically the two ingredient lists had the same functional headings but were labeled differently. As part of the next feature in import/exporting recipes with their respective nutrients, we would need a standardised set of keys for the nutrients - so I changed one ingredient list to match the keys of the other ingredient list. Obviously we could've had OR statements to check for the presence of either key but this can get quite verbose if we have ingredient list that follow different heading naming conventions - hence the approach to having a standard set of keys for containing nutritional info. If this isnt helpful hopefully the pseudocode below helps - 
import example:
```
for ingredient_line in csv:
ingredient_nutrient = { "C_DRYMATTER_G" : csv["C_DRYMATTER_G"], etc etc}
```

Reply:
Yeah makes sense, lovely!